ui/app/i18n/appointments/index.js

```diff
@@ -0,0 +1,6 @@
+import en from './locale_en.json';
+import es from './locale_es.json';
+import fr from './locale_fr.json';
+import ptBr from './locale_pt_BR.json';
+
+export default{en, es, fr, ptBr};
```

Comment:
A few questions on this one. 
 - Are we saying now translations need to be part of the bundle as well? We already have translations at 2 different places `base i18n folder(../i18n)` and `custom i18n folder(/bahmni_config/apps/i18n)`.
 - As of now, I see `en, es, fr, pt_BR` as the translations getting bundled. Will it be a code change now, if we want ot support a new languages (e.g. `it`)?